src/storage/v2/inmemory/storage.hpp

```diff
@@ -76,6 +76,10 @@ class InMemoryStorage final : public Storage {
     /// @throw std::bad_alloc
     VertexAccessor CreateVertex() override;
 
+    // TODO Better naming
+    /// @throw std::bad_alloc
+    VertexAccessor CreateVertexEx(storage::Gid gid);
```

Comment:
`CreateVertexEx` and `CreateEdgeEx` was protected for a reason.
It is very dangerous and should only be used by the replication logic.
There is a special `ReplicationAccessor` for use in replication that enables these functions.
Why do you need these functions?